applications/FluidDynamicsApplication/tests/Cavity/steady_cavity10_probe_ref_1.dat

```diff
@@ -1,3 +1,3 @@
 # Results for "element" with Id # 111 at position: x: 0.6; y: 0.6; z: 0
 # time VELOCITY_X VELOCITY_Y PRESSURE
-1.0 -0.09267061845199485 -0.01595209625236281 -1.9070853965451051
+1.0 -0.0926712882273655 -0.015953583117680845 -1.9072522039311568
```

Comment:
Why are these tests changing? Is it because of the relaxation? I would prefer keeping the old behavior for the test if at all possible.